submissions/paulElugbaju/database2/trans.sql

```diff
@@ -0,0 +1,35 @@
+CREATE DATABASE jibola_transport_limited;
+
+USE jibola_transport_limited;
+
+CREATE TABLE passengers (
+	id INT NOT NULL AUTO_INCREMENT,
+    full_name VARCHAR(255) NOT NULL,
+    sex ENUM ('male', 'female') NOT NULL DEFAULT 'male',
+    age VARCHAR(255),
+    PRIMARY KEY (id)
+);
+
+CREATE TABLE passengers_details (
+	id INT NOT NULL AUTO_INCREMENT,
+    passenger_id INT NOT NULL,
+    class ENUM ('1', '2', '3') NOT NULL DEFAULT '1',
+    ticket_number VARCHAR(255) NOT NULL,
+    trip_fare DECIMAL (10, 2) NOT NULL,
+    assigned_cabin VARCHAR(255),
+    parents_children_aboard VARCHAR(255) NOT NULL,
+    siblings_spouses_aboard VARCHAR(255) NOT NULL,
```

Comment:
The number of siblings/spouses aboard is what is required here, so could you change the datatype to INT?